README.md

```diff
@@ -13,54 +13,20 @@ This project provides an interface from Python to the [SCIP Optimization Suite](
 Documentation
 -------------
 
-Please consult the [online documentation](https://scipopt.github.io/PySCIPOpt/docs/html) or use the `help()` function directly in Python or `?` in IPython/Jupyter.
+Please consult the [online documentation](https://pyscipopt.readthedocs.io/en/latest/) or use the `help()` function directly in Python or `?` in IPython/Jupyter.
```

Comment:
Until we have all the methods and the pop-down menus and stuff, I think we still need to link to the old documentation

Reply:
I strongly disagree with this. I think the API is only a small fraction of the documentation, and I don't think pop-down menus are even necessary.

Reply:
It should be said: The old documentation will stay online, so people can continue to use it

Reply:
Yeah, but there are some things in the old documentation that seem important. The docstrings, for example, which explain what the arguments of the methods are. This is really the only page I think should be referenced: [Model class reference](https://scipopt.github.io/PySCIPOpt/docs/html/classpyscipopt_1_1scip_1_1Model.html)

Are you against using a sentence like

> We are currently migrating the documentation to [online documentation](https://pyscipopt.readthedocs.io/en/latest/). It should be prefered to the old documentation (can still be found in ...) which will be deprecated.

Reply:
You're completely correct on the docstrings......
I've added a strong sentence immediately after the current documentation link.